devenv/scripts/run_public_cluster.sh

```diff
@@ -0,0 +1,11 @@
+	set -e
+
+	echo "beginning public cluster tests..."
+	make clean
```

Comment:
more weird indentation

Reply:
looks fine on ide but got rid of indentation